octoprint_mrbeam/filemanager/__init__.py

```diff
@@ -21,6 +22,10 @@ class MrbFileManager(FileManager):
     MAX_HISTORY_FILES = 25  # TODO fetch from settings
     MAX_GCODE_FILES = 25  # TODO fetch from settings
 
+    FILE_EXTENSIONS_SVG = ["svg"]
+    FILE_EXTENSIONS_HISTORY = ["mrb"]
+    FILE_EXTENSIONS_GCODE = ["g", "gc", "gco", "gcode", "nc"]
+
```

Comment:
I think the more octoprinty way would be to declare those file extensions in a hook. Well, that is, when we have our slicer code wrapped in a nice hook for OP 1.4
See https://docs.octoprint.org/en/master/plugins/hooks.html?highlight=filemanager#octoprint-filemanager-extension-tree

Reply:
It can trigger different slicers depending on the files ( if I remember correctly )